src/commons_codec/transform/dynamodb.py

```diff
@@ -92,11 +97,13 @@ def sql_ddl(self):
         Define SQL DDL statement for creating table in CrateDB that stores re-materialized CDC events.
         """
         return (
-            f"CREATE TABLE IF NOT EXISTS {self.table_name} "
-            f"({self.TYPED_COLUMN} OBJECT(DYNAMIC), {self.UNTYPED_COLUMN} OBJECT(IGNORED));"
+            f"CREATE TABLE IF NOT EXISTS {self.table_name} ("
+            f"{self.PK_COLUMN} OBJECT(DYNAMIC) AS ({', '.join(self.primary_key_schema.to_sql_ddl_clauses())}), "
```

Comment:
Should we rather make the `pk_column` an `OBJECT(STRICT)`? The primary key cannot be changed on an existing table. Although it doesn't make a difference in practice, since the object isn't the primary key, but the elements inside. And an `OBJECT(STRICT)` also still can be change through an `ALTER TABLE ADD COLUMN`. It just looks a bit untypical to have a dynamic PK. Are there scenarios where new attributes would be added to the PK object during runtime?

Reply:
Updated.